tools/RELEASE.md

```diff
@@ -24,163 +24,319 @@ The following guide uses `v0.11.4rc1` as version for a release candidate,
 replace as necessary with your version or remove `rc..` suffix for the
 final release.
 
+To give you an idea what this process looks like we have this fabricated
+time line:
+
+ * Time to prepare for a release.
+ * A librdkafka release candidate is available.
+ * Check if any open PRs needs merging, do that.
```

Comment:
I found that there are 16 open pull requests https://github.com/confluentinc/confluent-kafka-python/pulls, is there a good way to distinguish which PRs need merging?